### PR TITLE
Please merge

### PR DESCRIPTION
**Pull request by deeptijk**

### CODE REVIEW

**Comment by ypriverol** [comment]
@deeptijk can you remove the prefix `PTMeXchange` on each name and folder. The structure should be: 

- Folder name: {PX accession}
- file name: {PX accession}.sdrf.tsv 
-

**Comment by deeptijk** [comment]
@ypriverol I finished all the validation. They all have fulfilled the plant template. Please merge them.

File: annotated-projects/PXD001774/PXD001774.sdrf.tsv [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Sample 1	Oryza sativa	Embryo	Plant cell	not applicable	not applicable	3	run 3	PXD001774	0HNP3timecourse.raw	http://ftp.pride.ebi.ac.uk/pride/data/archive/2016/03/PXD001774/0HNP3timecourse.raw	1	1	NT=label free sample	AC=MS:1000449;NT=LTQ Orbitrap	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	10 ppm	0.8 Da
+Sample 2	Oryza sativa	Embryo	Plant cell	not applicable	not applicable	1	run 4	PXD001774	12HNP1timecourse.raw	http://ftp.pride.ebi.ac.uk/pride/data/archive/2016/03/PXD001774/12HNP1timecourse.raw	1	1	NT=label free sample	AC=MS:1000449;NT=LTQ Orbitrap	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	10 ppm	0.8 Da
```
Comment (daichengxin) [inline annotated-projects/PXD001774/PXD001774.sdrf.tsv:5]: Biological replicate require different source name

Comment (deeptijk) [reply annotated-projects/PXD001774/PXD001774.sdrf.tsv]: I have changed it. Have a look.

Comment (deeptijk) [reply annotated-projects/PXD001774/PXD001774.sdrf.tsv]: @daichengxin Can you please check now and merge it.

File: annotated-projects/PXD005241/PXD005241.sdrf.tsv [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Sample 3	Oryza sativa	Leaf	Callus	Root	Shoot meristem	Young panicles	Mature panicles	not applicable	not applicable	3	run 18	 japonica	PXD005241	Rice_phos_young panicles__TiO2_25per_F3_R1.raw.raw	https://www.ebi.ac.uk/pride/archive/projects/PXD005241/Rice_phos_young panicles__TiO2_25per_F3_R1.raw.raw	1	1	NT=label free sample	AC=MS:1001911;NT=Q Exactive	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:50	NT=Acetyl;MT=Variable;PP=Protein N-term;AC=UNIMOD:16	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	not available	not available
```
Comment (daichengxin) [inline annotated-projects/PXD005241/PXD005241.sdrf.tsv:19]: The source name of different samples should be different.  The source name here should be Sample 1, Sample 2, Sample 3, Sample 4, Sample 5, Sample 6, Sample 7...

Comment (deeptijk) [reply annotated-projects/PXD005241/PXD005241.sdrf.tsv]: @daichengxin thanks, I have changed it.

**Comment by deeptijk** [comment]
@daichengxin Please can you check the new files I have uploaded.

File: annotated-projects/PXD000923/PXD000923.sdrf.tsv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Sample 9	Oryza sativa	Flower	not applicable	not applicable	2	run 17	GZ63S	PXD000923	LHY_T9.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	9	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
+Sample 10	Oryza sativa	Flower	not applicable	not applicable	2	run 19	GZ63S	PXD000923	LHY_T10.wiff	http://ftp.pride.ebi.ac.uk/pride/data/archive/2014/07/PXD000923/raw_data.zip	10	1	NT=label free sample	AC=MS:1000932;NT=TripleTOF 5600	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	15 ppm	0.05 Da
```
Comment (daichengxin) [inline annotated-projects/PXD000923/PXD000923.sdrf.tsv:11]: It seems that the first 5 files (run 1 - run 9 or run 11 - run 19)are from the same fraction group under a sample. So their source name should be the same.   Other annotations look good.

**Review by daichengxin — CHANGES_REQUESTED** [review]

File: annotated-projects/PXD000923/PXD000923.sdrf.tsv (hunk 3 above)
Comment (deeptijk) [reply annotated-projects/PXD000923/PXD000923.sdrf.tsv]: @daichengxin Please have a look into the datasets, At least 9 out of 10 looks ok, so if possible please merge it with the master. Thank you

**Pull request merged**